pioinstaller/__main__.py

```diff
@@ -29,14 +31,20 @@
 @click.group(name="main", invoke_without_command=True)
 @click.version_option(__version__, prog_name=__title__)
 @click.option("--verbose", is_flag=True, default=False, help="Verbose output")
+@click.option("--shutdown-servers", is_flag=True, default=True)
 @click.pass_context
-def cli(ctx, verbose):
+def cli(ctx, verbose, shutdown_servers):
     if verbose:
         logging.getLogger("pioinstaller").setLevel("DEBUG")
     log.debug("Invoke: %s", " ".join(sys.argv))
     log.debug("sys.platform: %s", sys.platform)
     log.debug("sys.version: %s", sys.version)
     log.debug("sys.executable: %s", sys.executable)
+    if shutdown_servers:
+        proc = multiprocessing.Process(target=shutdown_pio_home_servers)
+        proc.start()
+        proc.join(10)
+        proc.terminate()
```

Comment:
Make this a part of `shutdown_pio_home_servers`. In this place you should call only shutdown_pio_home_servers.

You can make `def _internal` of `def shutdown_pio_home_servers`